src/analysis/specific_limits.lean

```diff
@@ -930,3 +930,39 @@ tendsto_of_tendsto_of_tendsto_of_le_of_le'
     { refine mul_nonneg _ (inv_nonneg.mpr _); norm_cast; linarith },
     { refine (div_le_one $ by exact_mod_cast hn).mpr _, norm_cast, linarith }
   end
+
+/-!
+### Ceil and floor
+-/
+
+lemma tendsto_nat_floor_mul_div_at_top {a : ℝ} (ha : 0 ≤ a) :
+  tendsto (λ x, (⌊a * x⌋₊ : ℝ) / x) at_top (𝓝 a) :=
```

Comment:
Is it worth adding the variant with the `*` and `/` swapped and `𝓝 0` instead of `at_top` (assuming that's even true!)

Reply:
In the long run we should have a tactic that handles this kind of limit (like the one presented by Manuel Eberl for Isabelle), so for now I'd only add the limits I need.